webapp/app.py

```diff
@@ -165,34 +168,50 @@ def global_template_context():
             component_tabs_file.read(), Loader=yaml.FullLoader
         )
 
+    updated_features = {}
+    for feature in FEATURES_LIST[0]["features"]:
+        updated_features[feature["url"]] = feature["status"]
```

Comment:
What is this for?

Reply:
create a dictionary mapping URLs to status for the side navigation